### ISSUE TITLE
It appears that postgis database connection require username/password stored

### ISSUE THREAD

**Issue by aileenh** [pgRouting/pgRoutingLayer#44]
When using pgrouting layer plugin with QGIS it appears that it will only display databases in the pgRouting Layer panel if the username and password is stored in the connection. 

Can you confirm if my understanding is correct?

**Comment by cvvergara** [comment]
Yes that is correct due to the fact that pgRoutingLayer does not ask for user/password

**Comment by aileenh** [comment]
This is a bit of a security issue. Could the plugin not use the connection assuming the user has connect (and logged in) to the database via QGIS.
